plugin/src/test/java/org/refactoringminer/test/RefactoringPopulator.java

```diff
@@ -78,7 +74,7 @@ private static void prepareFSERefactorings(TestBuilder test, BigInteger flag)
     private static boolean isAdded(Refactoring refactoring, BigInteger flag) {
         try {
             BigInteger value = Enum.valueOf(Refactorings.class, refactoring.type.replace(" ", "")).getValue();
-            return value.and(flag).compareTo(BigInteger.ZERO) == 1;
+            return value.and(flag).compareTo(BigInteger.ZERO) > 0;
```

Comment:
what is the reason to change it? o_o

Reply:
Idea warning. It is documented for BigInteger that it is 1 but as I know it is not guaranteed in other cases so it is common practice to compare with 0